packages/oca/src/legacy/resolver/oca.ts

```diff
@@ -203,93 +160,129 @@ export class OCABundle implements OCABundleType {
     return this.getOverlay<MetaOverlay>(OverlayType.Meta10, this.options.language)
   }
 
-  public get cardLayoutOverlay(): CardLayoutOverlay10 | CardLayoutOverlay11 | undefined {
-    return this.getOverlay(this.options?.cardOverlayType || CardOverlayType.CardLayout11)
+  public get brandingOverlay(): BrandingOverlay | LegacyBrandingOverlay {
+    // making sure that it wont be undefined
```

Comment:
I'm trying here to avoid undefined branding overlays. This can happen if a bundle is defined but the branding overlay is not defined or it is defined, but the selected branding overlay type is other than the defined one. Any thoughts on this case?